src/profile/v100/app_preferences/gateways/saved_gateways.rs

```diff
@@ -145,53 +145,57 @@ impl Gateways {
     }
 }
 
-impl Default for Gateways {
+impl Default for SavedGateways {
     fn default() -> Self {
         Self::new_with_other(Gateway::mainnet(), vec![Gateway::stokenet()])
             .expect("Stokenet and Mainnet should have different NetworkIDs.")
     }
 }
 
-impl HasSampleValues for Gateways {
+impl HasSampleValues for SavedGateways {
     fn sample() -> Self {
-        let mut sut = Gateways::new(Gateway::rcnet());
+        let mut sut = SavedGateways::new(Gateway::rcnet());
```

Comment:
can you change to `let mut gateways = Self::new` (chaingin `sut` and `SavedGateways`.

(probably old copy paste error from me, calling it `sut`....)